docs/examples/module-temperature-check.py

```diff
@@ -0,0 +1,60 @@
+"""
+Module Temperature Check
+========================
+
+Test whether the module temperature is correlated with irradiance.
+"""
+
+# %%
+# Testing the correlation between module temperature and irradiance
+# measurements can help identify if there are issues with the module
+# temperature sensor.
+# In this example, we demonstrate how to use
+# :py:func:`pvanalytics.quality.weather.module_temperature_check`, which
+# runs a linear regression model of module temperature vs. irradiance. Model
+# performance is then assessed by correlation coefficient. If it meets a
+# minimum threshold, function outputs a True boolean. If not, it outputs a
+# False boolean.
+
+import pvanalytics
+from pvanalytics.quality.weather import module_temperature_check
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy.stats import linregress
+import pathlib
+
+# %%
+# First, we read in example data from the NREL SERF West system, which contains
+# data for module temperature and irradiance under the 'module_temp_1__781'
+# and 'poa_irradiance__771' columns, respectively. This data set contains
+# 15-minute averaged measurements, and is available via the NREL PVDAQ database
+# as system 51.
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+serf_east_file = pvanalytics_dir / 'data' / 'serf_west_15min.csv'
+data = pd.read_csv(serf_east_file, index_col=0, parse_dates=True)
+print(data[['module_temp_1__781', 'poa_irradiance__771']].head(10))
+
```

Comment:
I think it would be good practice to plot these data in addition to printing the head.

Reply:
Added plots for both of these in https://github.com/pvlib/pvanalytics/pull/151/commits/553bbe8e28e8216cba86c8ae7be117e93d80aae4